commitlint.config.ts

```diff
@@ -265,7 +281,7 @@ module.exports = {
 
                     // taken from https://stackoverflow.com/a/66433444/544947 and https://unix.stackexchange.com/a/25208/56844
                     let recommendedUnixCommand =
-                        'git log --format=%B -n 1 $(git log -1 --pretty=format:"%h") | cat - > log.txt ; fmt -w 1111 -s log.txt > ulog.txt && fmt -w 64 -s ulog.txt > wlog.txt && git commit --amend -F wlog.txt';
+                        'git log --format=%B -n 1 $(git log -1 --pretty=format:"%h") | cat - > log.txt ; fmt -w 1111 -s log.txt > ulog.txt && fmt -w 64 -s -u ulog.txt > wlog.txt && git commit --amend -F wlog.txt';
```

Comment:
I guess it's about removing extra spaces between the words

Reply:
in macOS, it already reduces for me any 2 spaces to 1, without your -u addition

Reply:
I see. However, in Linux, it doesn't remove extra spaces without the -u option.

Reply:
You're right! But in macOS it gives an error :( so let's improve the error message so that it shows both the Linux and the macOS command (and while you're at it, let's rebase the PR).

Reply:
Done